pubspec.yaml

```diff
@@ -11,7 +11,9 @@ dependencies:
   async: ^2.11.0
   flutter:
     sdk: flutter
-  flutter_key_value_file_storage: ^0.0.8
+  # TODO remove path dependency after merging: https://github.com/icapps/flutter_key_value_file_storage/pull/14
+  flutter_key_value_file_storage:
+    path: ../flutter_key_value_file_storage
```

Comment:
Package is merged, so this can be changed to ^1.0.0

Reply:
Changed in [e80794c](https://github.com/icapps/flutter_secure_file_storage/pull/33/commits/e80794ca3613ef6a88b311e33cca393c03f9fa8b)